src/commands/smite.ts

```diff
@@ -0,0 +1,96 @@
+import { EmbedBuilder, SlashCommandBuilder } from 'discord.js';
+import { isAdmin, setUserSmitten } from '../helpers/smiteUtils.js';
+import { UserMessageError } from '../helpers/UserMessageError.js';
+
+const builder = new SlashCommandBuilder()
+	.setName('smite')
+	.setDescription('Smite a user, preventing them from using bot commands')
+	.addUserOption(option =>
+		option.setName('user').setDescription('The user to smite').setRequired(true)
+	);
+
+const ODIN_SMITING_THOR_GIF =
+	'https://cdn.discordapp.com/attachments/781563734098575410/1083403217553084446/smite.gif?ex=68fb93df&is=68fa425f&hm=83b4a865179f0dd6fb1312f535c2a730673709734470c0095dd2a432908bfcba&';
+const WACK_IMAGE =
+	'https://i.kym-cdn.com/entries/icons/original/000/033/758/Screen_Shot_2020-04-28_at_12.21.48_PM.png';
```

Comment:
This link still works, but feels weird to rely on an external service for a file we may frequently access. We should provide this file directly as well

Reply:
I wonder if we should make a simple HTTP server in CSBot that serves static images like these ones :thinking: